solutions/4.js

```diff
@@ -0,0 +1,11 @@
+const jump = (str, num, newstr='', i=0)=>{
+  if (i==num){
+    return newstr;
+  }
+  console.log(str);
+  newstr = newstr + str;
```

Comment:
no need to create a new string. pass it in directly: `return jump(str, num, newstr + str, i+1);`